src/components/video-embed/docs.mdx

```diff
@@ -0,0 +1,61 @@
+---
+componentName: VideoEmbed
```

Comment:
I've added this to Swingset mainly just to expose existing documentation from the `learn` repo. These docs are definitely more author-oriented - I think this makes sense, since from a dev perspective `VideoEmbed` is a lightly styled wrapper around `react-player`.